### PR TITLE
Update ProjectLibraryPerspective according to AF-2177 changes

### PR DESCRIPTION
**Pull request by tomasdavidorg**

### CODE REVIEW

**Comment by tomasdavidorg** [comment]
Jenkins execute full downstream build

File: business-central-tests/business-central-tests-gui/src/main/java/org/kie/wb/selenium/model/persps/ProjectLibraryPerspective.java [hunk 1]
```diff
@@ -50,14 +52,23 @@ public boolean isDisplayed() {
     }
 
     private ImportRepositoryModal importProject() {
-        click(PROJECT_ACTIONS_BUTTON);
-        click(IMPORT_PROJECT_BUTTON);
+        if (Waits.isElementPresent(IMPORT_PROJECT_BUTTON)) {
```
Comment (jomarko) [inline business-central-tests/business-central-tests-gui/src/main/java/org/kie/wb/selenium/model/persps/ProjectLibraryPerspective.java]: Is this false when some projects already available, or is this new `if` check rather for random failure protection.

Comment (tomasdavidorg) [reply business-central-tests/business-central-tests-gui/src/main/java/org/kie/wb/selenium/model/persps/ProjectLibraryPerspective.java]: Yes. According to the latest changes if there is no project both buttons are present in the view:
image
if there is already some project, the buttons are under the actions dropdown
image

**Comment by ederign** [comment]
@jomarko we will merge this in order to have a green build over the weekend ok? If you need anything else we can change it on further PR's.

**Pull request merged**